MAINTAINERS.md

```diff
@@ -25,29 +99,57 @@ git push -u origin release-$now
 scripts/have-news HEAD^ > have-news.md
 ```
 
-Create a release PR, pasting `have-news.md` into the body.  Then build the SDK:
+- [ ] Create the release PR.
+
+The above should have pushed state to GitHub to let you create a
+new PR to merge the release branch back to the main branch. The PR
+name should follow the convention of previous releases, which is
+currently `chore(release): publish _release_label_`.  Paste
+`have-news.md` as the description of the PR.  Follow the example
+of previous releases for any other details of the PR.
+
+Creating this PR will also kick off the CI tests.
+
+- [ ] Build the NPM-installed SDK packages.
+
+While the above CI tests run, build the SDK:
 
 ```sh
 # Build all package generated files.
-yarn install
+yarn install --force
 yarn build
 ```
 
-Once tests pass, you can publish to NPM with the following:
+- [ ] Wait for the release PR's CI tests to pass.
+
+### Publish the Release
+
+These steps cannot be undone, so be sure that you are ready to proceed.
+In particular, be sure that you have waited for the release PR's CI tests
+to pass.
+
+- [ ] Publish to NPM
 
 ```sh
 # Publish to NPM. NOTE: You may have to repeat this several times if there are failures.
 yarn lerna publish from-package
 ```
 
-Merge the release PR into master.  DO NOT REBASE OR SQUASH OR YOU WILL LOSE
-REFERENCES TO YOUR TAGS.
+- [ ] Merge the release PR into the base branch.  DO NOT REBASE OR SQUASH OR YOU WILL LOSE
+REFERENCES TO YOUR TAGS. You may use the "Merge" button directly instead of automerge.
+
+- [ ] DO NOT change your local git environment to the base branch - keep
+it on the release branch for the following steps.
+
+- [ ] Publish the released package tags
```

Comment:
if you're feeling documentarian, this could add:
* this will push a `${package}@${version}` tag, one per package, plus the repo-wide `agoric-sdk@${version}` tag, plus the golang-specific `v${version}` tag (whose version matches the one used for `@agoric/cosmic-swingset`)
* in fact, it will push all tags that match these patterns, but all the old version's tags will already be present on GitHub

Reply:
also, I think it's the pushing of one of these tags which triggers the docker build, but don't quote me on that

Reply:
Added the description, but not the speculation.